ansible/roles/ocp-workload-velero/tasks/post_workload.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: "Installing ocpmigrate plugin"
+  shell: "{{ velero_download_dir }}/velero/velero plugin add quay.io/ocpmigrate/velero-plugin"
```

Comment:
Wonder if we can skip downloading velero release and just install plugin as below example does?

https://github.com/fusor/mig-controller/blob/master/hack/deploy/manifests/20-velero-deployment.yaml#L65-L74